src/compat/tracker.jl

```diff
@@ -12,6 +12,8 @@ using .Tracker: Tracker,
 using Compat: eachcol
 using LinearAlgebra
 
+import Base.*
+
```

Comment:
IMO readability and clarity is increased by implementing the overloaded functions as `function Base.:*(...)` and removing the import:
```suggestion
```

Reply:
Removed type piracy code.